lib/config.go

```diff
@@ -57,6 +57,24 @@ const (
 	DefaultLogSizeMax = -1
 )
 
+// DefaultCapabilities for the capabilities option in the crio.conf file
+var DefaultCapabilities = []string{
+	"CHOWN",
+	"DAC_OVERRIDE",
+	"FSETID",
+	"FOWNER",
+	"MKNOD",
+	"NET_RAW",
+	"SETGID",
+	"SETUID",
+	"SETFCAP",
+	"SETPCAP",
+	"NET_BIND_SERVICE",
+	"SYS_CHROOT",
+	"KILL",
+	"AUDIT_WRITE",
```

Comment:
I propose we remove AUDIT_WRITE from defaults.
/* Allow writing the audit log via unicast netlink socket */

This allows an application to send a message to the audit log system, I think this should be blocked by default.